sdk/core/Azure.Core.Experimental/src/Azure.Core.Experimental.csproj

```diff
@@ -5,7 +5,7 @@
     <Version>0.1.0-preview.38</Version>
     <PackageTags>Microsoft Azure Client Pipeline</PackageTags>
     <Nullable>enable</Nullable>
-    <TargetFrameworks>$(RequiredTargetFrameworks);net462;net6.0</TargetFrameworks>
+    <TargetFrameworks>$(RequiredTargetFrameworks);net462;net8.0</TargetFrameworks>
```

Comment:
RequiredTargetFrameworks includes net8.0 unless there is an override. So this net6.0 target can just be removed.